implementations/rust/ockam/ockam_command/src/project/enroll.rs

```diff
@@ -177,6 +176,14 @@ async fn run_impl(
         .overwrite(&project.name, project.clone().try_into()?)?;
 
     let credential = client2.credential().await?;
+    let identity = identities()
+        .identities_creation()
+        .decode_identity(authority.identity())
+        .await?;
+    opts.state.credentials.overwrite(
```

Comment:
is this  per-identity?,    I mean, if the node has two identities,   credential for `i1`  would not override credential for `i2` right?

Reply:
It's storing the credential using the project's authority identity as the issuer and the project's name as the file name (kind of the "unique id" in the file system). Should we use another name for the file to avoid false duplicates? (the identity name as you mentioned perhaps?)

Reply:
Maybe we should wait until this is implemented
https://github.com/build-trust/ockam/issues/4952